easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2019b.eb

```diff
@@ -0,0 +1,71 @@
+# easybuild easyconfig
+#
+# Graphviz has moved to gitlab and it is no longer possible to get tagged
+# versions.  There is an open ticket this issue.
+
+easyblock = 'ConfigureMake'
+
+name = 'Graphviz'
+version = '2.40.1'
+
+homepage = 'http://www.graphviz.org/'
+description = """Graphviz is open source graph visualization software. Graph
+ visualization is a way of representing structural information as diagrams of
+ abstract graphs and networks. It has important applications in networking,
+ bioinformatics,  software engineering, database and web design, machine
+ learning, and in visual interfaces for other technical domains."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES']
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
This does not work. The URL for the official download is http://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/graphviz.tar.gz. The problem is that it has always lacked any versioning, so there is no way to keep track of changes to it.